desktop/integration-test/websocket.test.ts

```diff
@@ -0,0 +1,48 @@
+// SPDX-FileCopyrightText: Copyright (C) 2023-2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)<lichtblick@bmwgroup.com>
+// SPDX-License-Identifier: MPL-2.0
+
+import { launchApp } from "./launchApp";
+import { launchWebsocketTest } from "./launchWebsocketTest";
+
+describe("websocket connection", () => {
+  it("should show correct attributes using custom test and raw messages panel", async () => {
+    const websocketServer = launchWebsocketTest();
+
+    const app = await launchApp();
+
+    await app.renderer.getByText("Open connection").click();
+    await app.renderer.getByText("Open", { exact: true }).click();
+
+    // Show connection to "ws://localhost:8765 websocket-test-server", it is located on top bar
+    await expect(
+      app.renderer.getByText("ws://localhost:8765 websocket-test-server").innerHTML(),
+    ).resolves.toBeDefined();
+
+    // Check if system is listed on topics menu
+    await app.renderer.getByText("Topics", { exact: true }).click();
+    await expect(app.renderer.getByText("/websocket_test").innerHTML()).resolves.toBeDefined();
+
+    // Add raw messages panel to check messages
+    await app.renderer.getByLabel("Add panel button").click();
+    await app.renderer.getByText("Raw Messages").click();
+
+    // Select the topic
+    await app.renderer.getByPlaceholder("/some/topic.msgs[0].field").nth(0).click();
+    await app.renderer.getByTestId("autocomplete-item").click();
+
+    const rawMessagesPanel = app.renderer.getByTestId(/RawMessages/);
+
+    // Check if message is correctly beeing displayed
+    const attributesToCheck = ["hello", '"world"', "foo", "42"];
+
+    for (const attribute of attributesToCheck) {
+      await expect(
+        rawMessagesPanel.getByText(attribute, { exact: true }).innerText(),
+      ).resolves.toBe(attribute);
+    }
+
+    await app.main.close();
+
+    void websocketServer.close();
+  }, 20_000);
```

Comment:
I don't know if having a huge test like that and increasing the time running to 20 seconds would be nice. 
 Locally I wasn't able to run all tests by using yarn run test:integration command. Maybe the reason why is that this test is too big and it runs first making other tests fail (there are running fine on main branch).
 My suggestion is trying to divide it in more tests / simplify it a bit so we can save some processing time either locally or in the pipeline.

Reply:
I reduce to 15s. It should take less than that, it is a max timeout. I think the default of 5s is too short, but it should finish faster